k8s/envs/local/secrets.env

```diff
@@ -0,0 +1,8 @@
+DB_PASSWORD=changeme
```

Comment:
I would add a `.gitignore` file to this directory with a `secrets.env` entry, and rename this file to `secrets.env.changeme`.

Sooner or later, someone will commit and push their keys to Github. And the only way to undo that is to rotate all keys. Github is like a immutable db, so once you push something it's accessible for ever https://trufflesecurity.com/blog/anyone-can-access-deleted-and-private-repo-data-github